src/hardware/sblaster.cpp

```diff
@@ -2215,6 +2206,24 @@ class SBLASTER final {
 		if (sb.type == SBT_NONE || sb.type == SBT_GB)
 			return;
 
+		sb.hw.dma8 = static_cast<uint8_t>(section->Get_int("dma"));
+		auto dma_channel = DMA_GetChannel(sb.hw.dma8);
+		assert(dma_channel);
+		dma_channel->ReserveFor(CardType(), SBLASTER_ShutDown);
+
+		// Only Sound Blaster 16 uses a 16-bit DMA channel.
+		if (sb.type == SB_TYPES::SBT_16) {
+			sb.hw.dma16 = static_cast<uint8_t>(section->Get_int("hdma"));
+
+			// Reserve the second DMA channel only if it's unique.
+			if (sb.hw.dma16 != sb.hw.dma8) {
+				dma_channel = DMA_GetChannel(sb.hw.dma16);
+				assert(dma_channel);
+				dma_channel->ReserveFor(CardType(),
+										SBLASTER_ShutDown);
```

Comment:
formatting

Reply:
I can't see the problem (maybe I'm missing something?)

![2023-06-19_15-23](https://github.com/dosbox-staging/dosbox-staging/assets/1557255/39494501-a47b-43de-95f3-f5dd3e023db1)

Reply:
That's what I see on my screen:

<img width="852" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/5ce244d7-fe79-4cac-887e-54292cc55c99">

Reply:
Oh, thanks - got it now 👍